documentation/compatibility_with_puppet_agent.markdown

```diff
@@ -1,159 +1,130 @@
-    * Note that `certificate*` endpoints are not validated via "auth.conf" rules
-      when `jruby-puppet.use-legacy-auth-conf` is not set or set to `true`.
-      If you need to configure authorization for these endpoints, consider
-      configuring this setting to `false`.  See the
-      [Puppet Server auth.conf](./configuration.markdown#authconf) page for
-      more information.
-* Add the rules to the appropriate "auth.conf" file on your Puppet Server.
+    * Note that `certificate*` endpoints are not validated by `auth.conf` rules when `jruby-puppet.use-legacy-auth-conf` is not set or set to `true`. If you need to configure authorization for these endpoints, consider configuring this setting to `false`. See the [Puppet Server `auth.conf` documentation][] for more information.
+* Add the rules to the appropriate `auth.conf` file on your Puppet Server.
 
 For more information, see:
 
-* [Puppet's HTTPS API (current)](/puppet/latest/reference/http_api/http_api_index.html)
-* [Puppet's HTTPS API (3.x)](https://github.com/puppetlabs/puppet/blob/3.8.0/api/docs/http_api_index.md)
-* [Default Puppet 4.1.0 auth.conf](https://github.com/puppetlabs/puppet/blob/4.1.0/conf/auth.conf)
-* [Default Puppet 3.8.0 auth.conf](https://github.com/puppetlabs/puppet/blob/3.8.0/conf/auth.conf)
-
-#### auth.conf Rule Example
-
-Puppet 3 rules, using the legacy Puppet "auth.conf" format:
-
-    # Puppet 3 auth.conf on the master
-    path ~ ^/catalog/([^/]+).uuid$
-    method find
-    allow /^$1\.uuid.*/
-
-    # Default rule, should follow the more specific rules
-    path ~ ^/catalog/([^/]+)$
-    method find
-    allow $1
-
-When `jruby-puppet.use-legacy-auth-conf` is `false`, Puppet Server 2 rules in
-`/etc/puppetlabs/puppetserver/conf.d/auth.conf` supporting both 3.x and 4.x
-agent nodes:
-
-    authorization: {
-        version: 1
-        rules: [
-            ...
-            {
-              # Puppet 3 & 4 compatible auth.conf with Puppet Server 2.2+
-              match-request: {
+* [Puppet 4 HTTPS API documentation](/puppet/latest/reference/http_api/http_api_index.html)
+* [Puppet 3 HTTPS API documentation](/references/3.8.0/developer/file.http_api_index.html)
+* [Puppet 4 `auth.conf` documentation][auth.conf]
+* [Puppet 3 `auth.conf` documentation](/puppet/3.8/reference/config_file_auth.html)
+
+#### Example `auth.conf` Rules for Puppet 3 and 4 Agents
+
+The other examples in this section convert this Puppet 3 example `auth.conf` rule to be compatible with Puppet 4:
+
+~~~
+# Puppet 3 auth.conf on the master
+path ~ ^/catalog/([^/]+).uuid$
+method find
+allow /^$1\.uuid.*/
+
+# Default rule, should follow the more specific rules
+path ~ ^/catalog/([^/]+)$
+method find
+allow $1
+~~~
+
+To support both Puppet 3 and Puppet 4 agents when the `use-legacy-auth-conf` parameter in the `jruby-puppet` setting is false, modify the rules to follow the new HOCON `auth.conf` format and place the new rules in `/etc/puppetlabs/puppet/conf.d/auth.conf`:
```

Comment:
The path reference at the end here should be `/etc/puppetlabs/puppetserver/conf.d/auth.conf` (adding "server" in the middle).